nilearn/datasets/func.py

```diff
@@ -1779,49 +1779,112 @@ def fetch_bids_langloc_dataset(data_dir=None, verbose=1):
 
 
 @fill_doc
-def fetch_openneuro_dataset_index(data_dir=None,
-                                  dataset_version='ds000030_R1.0.4',
-                                  verbose=1):
+def fetch_openneuro_dataset_index(
+    data_dir=None,
+    dataset_version='ds000030_R1.0.4',
+    verbose=1,
+):
     """Download a file with OpenNeuro :term:`BIDS` dataset index.
 
+    .. deprecated:: 0.9.1
+        `fetch_openneuro_dataset_index` will be removed in 0.13.
+
     Downloading the index allows to explore the dataset directories
     to select specific files to download. The index is a sorted list of urls.
 
     Parameters
     ----------
     %(data_dir)s
-    dataset_version : string, optional
+    dataset_version : :obj:`str`, optional
         Dataset version name. Assumes it is of the form [name]_[version].
         Default='ds000030_R1.0.4'.
+
+        .. warning:: Any value other than the default will be ignored.
+
+    %(verbose)s'
+
+    Returns
+    -------
+    urls_path : :obj:`str`
+        Path to downloaded dataset index.
+    urls : :obj:`list` of :obj:`str`
+        Sorted list of dataset directories.
+    """
+    warnings.warn(
+        (
+            'The "fetch_openneuro_dataset_index" function was deprecated in '
+            'version 0.9.1, and will be removed in 0.13. '
```

Comment:
I thought we were using 2 minor release cycles for deprecation length. I was expecting `0.11` since we deprecate in the 0.9.x series. It is not super costly to keep it until `0.13` though if you think that's better.

Reply:
Ah, right. I think I just blanked on the cycle length. Thanks for catching it.